sanoid

```diff
@@ -130,7 +130,7 @@ sub monitor_snapshots {
 
 	my ($config, $snaps, $snapsbytype, $snapsbypath) = @_;
 	my %datestamp = get_date();
-	my $errorlevel = 0;
+	my $errlevel = 0;
```

Comment:
Both forms are used in this file and there didn't seem to be any particular reason to use two different tokens for the same task. I personally would favor the longer form, but the shorter form was used more and feels more in keeping w/ the rest of the file/lines.